### DISCUSSION TITLE
Serve static assets with an efficient cache policy

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#66131] [Help] [answered]
### Summary

How could I fix this error? 

![image](https://github.com/vercel/next.js/assets/93617433/5acec45a-519d-4ae7-b48a-8f250d596a0c)

^^ It shows up on all of my pages

**Comment by icyJoseph** [comment]
What resource was it?

**Reply by Elindo586** [reply]
What do you mean resource?  The code?
The code repo is this below... it's showing up on all the pages with the same message
https://github.com/Elindo586/nextprojectb

**Reply by icyJoseph** [reply, marked as answer]
There, in the report, it should show a url, that's the resource that doesn't have an efficient cache policy

**Reply by Elindo586** [reply]
I got you..
![image](https://github.com/vercel/next.js/assets/93617433/5244c374-f2f2-4574-aa44-a5955cd6f6fe)


For example one link would be this:
https://www.google.com/maps/vt?pb=!1m5!1m4!1i8!2i68!3i94!4i256!2m3!1e0!2sm!3i693439329!2m3!1e2!2sspotlit!5i1!3m12!2sen!3sUS!5e289!12m4!1e68!2m2!1sset!2sRoadmap!12m3!1e37!2m1!1ssmartmaps!4e0!5m1!5f2!27m23!299174093m22!14m21!1m16!1m2!1y9810157407563562111!2y1835405012784659388!2zL20vMHZnYm0!4m2!1x424989936!2x3461290128!8b1!12m6!1m2!1x424389420!2x3460591757!2m2!1x425293700!2x3461799046!15sgcid%3Alocality!2b0!3b1!6b0!8b0&client=google-maps-embed&token=419


I think I got the error on all pages because I was running the lighthouse report from the localhost..

When I run the lighthouse from incognito, in the live site, then I don't get those errors on all pages.

I am guessing the error came from the images from each page.. it gives me the error in the localhost, but not in the live site.

What should be the proper thing to do with the error from the Google images from the imbeds?   It seems silly to try to add the images to the public folder.. 


Oh and now I can't upload the site to Vercel because it's broken...

**Reply by icyJoseph** [reply]
I mean, then we are back again at Google giving you headaches by giving you suboptimal code/resources, but also creating a tool that complains about them 🤷‍♂️
